src/BylawsText.js

```diff
@@ -0,0 +1,297 @@
+import React from 'react'
+import join from 'join-array'
+
+export default function BylawsText({ answers }) {
+  const answersArray = Object.entries(answers)
+  const anyAnswerGiven = answersArray.some(([_, answer]) => {
+    const isAnswerCheckbox = Array.isArray(answer)
+    return isAnswerCheckbox
+      ? answer.filter(Boolean).length > 0
+      : answer.length > 0
+  })
+
+  console.log('answers:', answers)
+
+  const nameAndLocationParagraph = (
+    <section id="nameAndLocation">
+      <h3>§ 1 Name und Sitz des Vereins</h3>
+      <p>
+        Der Name des Vereins lautet {answers.clubName} e.V. Der Verein hat
+        seinen Sitz in {answers.clubLocation}.
+      </p>
+    </section>
+  )
+
+  const businessYearParagraph = (
+    <section id="businessYear">
+      <h3>§ 2 Geschäftsjahr</h3>
+      <p>Geschäftsjahr ist das Kalenderjahr.</p>
+    </section>
+  )
+
+  const clubPurposeParagraph = (
+    <section id="clubPurpose">
+      <h3>§ 3 Zweck des Vereins</h3>
+      <p>
+        Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke
+        im Sinne des Abschnitts "Steuerbegünstigte Zwecke" der Abgabenordnung.
+        Zweck des Vereins ist {answers.clubPurpose}.
+      </p>
+    </section>
+  )
+
+  const nonProfitParagraph = (
+    <section id="nonProfit">
+      <h3>§ 4 Gemeinnützigkeit</h3>
+      <p>
+        Der Verein ist selbstlos tätig. Er verfolgt nicht in erster Linie
+        eigenwirtschaftliche Zwecke.
+      </p>
+    </section>
+  )
+
+  const useOfFundsParagraph = (
+    <section id="useOfFunds">
+      <h3>§ 5 Mittelverwendung</h3>
+      <p>
+        Mittel des Vereins dürfen nur für die satzungsmäßigen Zwecke verwendet
+        werden. Die Mitglieder erhalten keine Zuwendungen aus Mitteln des
+        Vereins.
+      </p>
+    </section>
+  )
+
+  const membersParagraph = () => {
+    const members = join({
+      array: answers?.typeOfMembers?.filter(Boolean),
+      last: ' und ',
+    })
+
+    const typeOfRequest = join({
+      array: answers?.typeOfRequest?.filter(Boolean),
+      last: ' oder ',
+    })
+
+    return (
+      <section id="members">
+        <h3>§ 6 Erwerb der Mitgliedschaft</h3>
+        <p>
+          {members} können Vereinsmitglieder werden. Der Aufnahmeantrag kann{' '}
+          {typeOfRequest} gestellt werden. Über den Aufnahmeantrag entscheidet
+          der Vorstand.Gegen die Ablehnung, die keiner Begründung bedarf, steht
+          dem/der Bewerber/in die Berufung an die Mitgliederversammlung zu,
+          welche dann endgültig entscheidet.
+        </p>
+      </section>
+    )
+  }
+
+  const admissionFeeParagraph = () => {
```

Comment:
Some components are functions and others just JSX code. Maybe make all of them into functional components?

Reply:
good point :-) done